.github/workflows/test.yml

```diff
@@ -14,22 +14,55 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
-  build:
+  test-flow:
+    runs-on: ubuntu-latest
+    steps:
+        # Checkss out the repository under $GITHUB_WORKSPACE
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Load Node
+        uses: actions/setup-node@v1
+      - name: Npm Install
+        run: npm install
+      - name: Flow type check
+        run: npm run flow
+  test-eslint:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        eslint: [3, 4, 5, 6]
```

Comment:
I'd probably skip running multiple versions of ESLint as sometimes the rules change between versions. I think you already have the Babel step for transpile for compatibility for old nodes, so you might just need a simple one version matrix like the previous job in the file